src/Simplex/Chat/Protocol.hs

```diff
@@ -331,6 +331,7 @@ data ChatMsgEvent (e :: MsgEncoding) where
   XGrpInv :: GroupInvitation -> ChatMsgEvent 'Json
   XGrpAcpt :: MemberId -> ChatMsgEvent 'Json
   XGrpLinkInv :: GroupLinkInvitation -> ChatMsgEvent 'Json
+  XGrpLinkAcpt :: MemberId -> ChatMsgEvent 'Json
```

Comment:
event not needed? group admittance process can continue from same statuses as before